apps/base-docs/docs/tools/block-explorers.md

```diff
@@ -41,6 +41,12 @@ A testnet explorer for [Base Goerli](https://base-goerli.dex.guru/) is also avai
 
 ---
 
+## L2scan Explorer
+
+[L2scan Explorer](https://base.l2scan.co/) is a web-based tool that allows users to explore and analyze the contents of the layer 2 blockchain networks like Base. It provides a user-friendly interface for viewing transaction history, checking account balances, and tracking the status of network activity.
```

Comment:
Can we change the first sentence to:
```
[L2scan Explorer](https://base.l2scan.co/) is a web-based tool that allows users to analyze Base and other layer 2 networks.
```

**Reason:** Thought "of the layer 2 blockchain networks" could be simplified, and that it could be more explicit that this is first and foremost for analyzing Base.